index.bs

```diff
@@ -648,23 +665,18 @@ ReadableStream(<var>underlyingSource</var> = {}, { <var>size</var>, <var>highWat
        initiate further reads from _reader_ or writes to _writer_. (Ongoing reads and writes may finish.) In particular,
        the user agent must check the below conditions on *this*.[[state]] and _dest_.[[state]] before performing any
        reads or writes, since they might lead to immediate shutdown.
```

Comment:
I think because of #644 I want to clarify what statuses are checked for at what time. Something like

* readable "ERRORED" and "CLOSED" trigger shutdown before a read
* readable "ERRORED" triggers shutdown before a write
* writable "CLOSING" and "CLOSED" are only checked at the beginning
* writable "ERRORED" triggers shutdown immediately at any time

Maybe the answer is to split out the "when to trigger" part from the "what to do" part.